lib/internal/fs/glob.js

```diff
@@ -376,6 +408,194 @@ class Glob {
       }
     }
   }
+
+
+  async* glob() {
+    ArrayPrototypePush(this.#queue, { __proto__: null, path: '.', patterns: this.#patterns });
```

Comment:
Do we really want/need primordials here as it builds on a dependency that doesn't use them?

Reply:
seems harfull